lib/Suggestions.js

```diff
@@ -12,6 +25,14 @@ class Suggestions extends Component {
     classNames: React.PropTypes.object
   }
 
+  componentDidUpdate = prevProps => {
+    const suggestionsContainer = this.refs.suggestionsContainer
+    if (suggestionsContainer && prevProps.selectedIndex !== this.props.selectedIndex) {
+      const activeSuggestion = suggestionsContainer.querySelector('.active')
+      activeSuggestion && maybeScrollSuggestionIntoView(activeSuggestion, suggestionsContainer)
```

Comment:
While this is very pithy, I think it would be clearer to just say 
```
if(activeSuggestion) {
    maybeScrollSuggestionIntoView(activeSuggestion, suggestionsContainer)
}
```